addons/website_event_sale/models/sale_order.py

```diff
@@ -78,21 +73,20 @@ def _cart_update(self, cr, uid, ids, product_id=None, line_id=None, add_qty=0, s
                 'event': ticket.event_id.name}
             new_qty, set_qty, add_qty = ticket.seats_available, ticket.seats_available, 0
 
-        values.update(super(sale_order, self)._cart_update(
-            cr, uid, ids, product_id, line_id, add_qty, set_qty, context, **kwargs))
+        values.update(super(SaleOrder, self)._cart_update(product_id, line_id, add_qty, set_qty, **kwargs))
 
         # removing attendees
         if ticket and new_qty < old_qty:
             attendees = Attendee.search(
-                cr, uid, [
+                [
```

Comment:
Not really necessary, but this bracket opening could probably move up to the next line